assets/constants.ts

```diff
@@ -6702,6 +6702,24 @@ export default {
         }
       ]
     },
+    "CURRICULUM_ADMIN": {
```

Comment:
Hi @Vir-8 just curious why this is **CURRICULUM_ADMIN** instead of **CLASSROOM_ADMIN**?
Is it a typo or any specific reason?

Thanks

Reply:
Hey, I think it was because curriculum admin is what we use elsewhere in the codebase even though the route is classroom admin - mainly the [isCurriculumAdmin](https://github.com/oppia/oppia/blob/e57a95b8eab30060273a818a9b3e7d039ca79c43/core/templates/domain/user/user-info.model.ts#L99) check. Also matches the name to be used from the main issue. Other than that, they're pretty synonymous I believe.